app/uk/gov/hmrc/trackingconsentfrontend/views/ErrorTemplate.scala.html

```diff
@@ -15,14 +15,14 @@
  *@
 
 @import uk.gov.hmrc.trackingconsentfrontend.config.AppConfig
-@(pageTitle: String, heading: String, message: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
-
-@contentHeader = {
-  <h1>@heading</h1>
-}
 
-@mainContent = {
-  <p>@message</p>
-}
-
-@govuk_wrapper(appConfig = appConfig, title = pageTitle, contentHeader = Some(contentHeader), mainContent = mainContent)
+@this(
+ layout: Layout,
+ paragraph: Paragraph,
+ h1: H1
+)
+@(pageTitle: String, heading: String, message: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@layout(pageTitle = Some(pageTitle)) {
+ @h1(content = heading)
+ @paragraph(content = Html(message))
+}
```

Comment:
Are you anticipating re-using these views? Having separate templates for h1 and paragraph seems a little odd as they're only used once so far.

You can also set reusable snippets inside the same template like this: https://www.playframework.com/documentation/2.8.x/ScalaTemplates#Declaring-reusable-blocks

Reply:
Yes, I have a follow-on PR with the static content that uses them.

Reply:
They are also used by the error template.